Makefile

```diff
@@ -645,12 +647,19 @@ syslog-image: syslog-build
 
 .PHONY: mock-grpc-server-image
 mock-grpc-server-image: mock-grpc-server-build clean-image
-	cp bin/linux_$(GOARCH)/mock-grpc-server integration-tests/mock-grpc-server/image/bin/mock-grpc-server
-	docker build \
+	cp -R bin integration-tests/mock-grpc-server/image
+	docker buildx build --platform $(PLATFORM) --load \
 		-t stackrox/grpc-server:$(TAG) \
 		-t quay.io/rhacs-eng/grpc-server:$(TAG) \
 		integration-tests/mock-grpc-server/image
 
+.PHONY: mock-grpc-server-image-push
+mock-grpc-server-image-push:
+	cp -R bin integration-tests/mock-grpc-server/image
+	docker buildx build --platform $(PLATFORM) --push \
```

Comment:
Will this just build and push for amd64 currently? I think we're only building multi-arch binaries but not multi-arch images

Reply:
As mentioned [here](https://github.com/stackrox/stackrox/pull/7482#discussion_r1297230267) only mock-grpc-server-image-push can be multi-arch.  I left the default PLATFORM as x86 in order to build the binaries and push the containers as multi-arch the PLATFORM needs to set in the environment.  I changed the build.yaml to refect this (NOTE: I'm not sure how to test build.yaml... it is untested).

Reply:
Just noticed I should probably add a target dependency for push to ensure we have binaries:

ie. 
.PHONY: mock-grpc-server-image-push
mock-grpc-server-image-push: mock-grpc-server-build

Reply:
build.yaml is running for this PR and appears to be green currently: https://github.com/stackrox/stackrox/actions/runs/5956150482/job/16156309812?pr=7482